### ISSUE TITLE
Create a Wiki page for contributors

### ISSUE THREAD

**Issue by piercus** [nervgh/angular-file-upload#739]
Create a wiki page for contribution guidelines

**Comment by piercus** [comment]
I will follow this https://help.github.com/articles/setting-guidelines-for-repository-contributors/

**Comment by DevNitish** [comment]
Documentation is not that helpful for a new developer. I am trying to use it with Node's npm package Multer. But it's hard to follow. if anyone can help here that would great.

**Comment by nervgh** [comment]
@piercus :+1: 

@DevNitish 

I think this issue it not about Node.js and Multer.
Did you use [search](https://github.com/nervgh/angular-file-upload/issues?utf8=%E2%9C%93&q=node.js) already?

**Comment by piercus** [comment]
#740 has been merged, i'm closing this one

**Comment by piercus** [comment]
@DevNitish it would be good if you can open a very specific issue about node.js and Multer.

If you have a specific issue with Multer, it will be very useful if you can help us to study it.

Please follow the [Contribution guidelines](https://github.com/nervgh/angular-file-upload/blob/master/CONTRIBUTING.md)
